example/rhizome/service/03230ce7-d8cb-410e-919e-19c480c1dd75.ttl

```diff
@@ -14,6 +14,8 @@
     core:hasPublisher "OKP4" ;
     core:hasTag "stockage" ;
     core:hasTitle "Clever Cloud Data Storage"@en .
+    core:hasTitle "Clever Cloud Data Storage"@fr .
```

Comment:
Syntax is not correct. Ending lines with `;` is required, except for the final line, which should end with `.`.

(this was detected by the CI but the error is not correctly reported)

Reply:
Thx I did the change.